src/Modal/index.js

```diff
@@ -186,12 +190,16 @@ const ModalWrapperContent = styled.div`
     overflow: visible;
     ${StyledModalSection}:last-of-type {
       padding-bottom: ${({ theme }) => theme.orbit.spaceXXLarge};
-      margin-bottom: ${({ fixedFooter, footerHeight }) =>
-        fixedFooter ? `${footerHeight}px` : "0"};
+      margin-bottom: ${({ hasModalSection, fixedFooter, footerHeight }) =>
+        hasModalSection && fixedFooter ? `${footerHeight}px` : "0"};
```

Comment:
`hasModalSection` is unnecessary condition in this case because you are styling `${StyledModalSection}:last-of-type` so the evaluation will be always true